elpis/wrappers/objects/dataset.py

```diff
@@ -194,11 +212,31 @@ def process(self):
             for d in temporary_directories:
                 os.rmdir(d)
 
+        # Reset the target corpus file so re-processing doesn't append
+        with open(self.pathto.corpus_txt, "w") as file_:
+            file_.truncate(0)
```

Comment:
This might be neater: `open(filename, 'w').close()`

Reply:
Thanks, much neater!